app/models/repository/filesystem.rb

```diff
@@ -60,4 +67,34 @@ def fetch_changesets
     nil
   end
 
+  private
+
+  def self.whitelisted_paths
+    OpenProject::Configuration["scm_filesystem_path_whitelist"]
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.